### PR TITLE
feat: Nethermind configs

### PR DESCRIPTION
**Pull request by Wolmin**
This PR provides configs for Nethermind client - both chainspec files and client configurations

### CODE REVIEW

File: mainnet/nethermind/nethermind.cfg [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "BaseDbPath": "./mainnet-data/execution"
+  },
+  "Sync": {
+    "SnapSync": true
+  },
+  "JsonRpc": {
+    "Host": "localhost",
+    "Port": 8551,
+    "Enabled": true,
+    "EnabledModules": ["eth","subscribe","trace","txpool","web3","personal","proof","net","parity","health","rpc","engine"],
```
Comment (mxmar) [inline mainnet/nethermind/nethermind.cfg:14]: Please change Nethermind defaults to our custom setup: `eth, engine, web3, net`.

Comment (Wolmin) [reply mainnet/nethermind/nethermind.cfg]: Resolved

**Review by mxmar — DISMISSED** [review]
LGTM @Wolmin 

maybe we can ask `Nethermind` for additional review :+1:

File: mainnet/nethermind/nethermind.cfg [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "JwtSecretFile": "./configs/shared/secrets/jwt.hex"
+  },
+  "Discovery": {
+    "Bootnodes": "enode://c2bb19ce658cfdf1fecb45da599ee6c7bf36e5292efb3fb61303a0b2cd07f96c20ac9b376a464d687ac456675a2e4a44aec39a0509bcb4b6d8221eedec25aca2@34.147.73.193:30303,enode://276f14e4049840a0f5aa5e568b772ab6639251149a52ba244647277175b83f47b135f3b3d8d846cf81a8e681684e37e9fc10ec205a9841d3ae219aa08aa9717b@34.32.192.211:30303"
+  },
+  "Merge": {
+    "Enabled": true
+  },
+  "KeyStore": {
+    "KeyStoreDirectory": "./testnet-data/execution/keystore"
```
Comment (mxmar) [inline mainnet/nethermind/nethermind.cfg:24]: @Wolmin Please change to `mainnet`

File: mainnet/shared/chainspec.json [hunk 3]
```diff
@@ -0,0 +1,889 @@
...
+    "eip2028Transition": "0x0",
+    "eip2200Transition": "0x0",
+    "eip2565Transition": "0x0",
+    "eip2929Transition": "0x0",
+    "eip2930Transition": "0x0",
+    "eip1559Transition": "0x0",
+    "eip3198Transition": "0x0",
+    "eip3529Transition": "0x0",
+    "eip3541Transition": "0x0",
+    "terminalTotalDifficulty": "0x0",
```
Comment (mxmar) [inline mainnet/shared/chainspec.json:49]: Please verify if there is `terminalTotalDifficultyPassed` flag available in Nethermind

@Wolmin Please consider moving to `/mainnet/nethermind/chainspec.json`

Comment (Wolmin) [reply mainnet/shared/chainspec.json]: I cannot see the passed option, so I assume having a null value replaces that

**Review by mxmar — APPROVED** [review]

**Pull request merged**